app/app/urls.py

```diff
@@ -271,6 +271,7 @@
 
     #passport
     re_path(r'^passport/$', passport.views.index, name='passport_gen'),
+    re_path(r'^passport-vc$', passport.views.verifiable_credential, name='passport_vc'),
```

Comment:
could we rename it tp passport/verify-credentials

Reply:
Resolved in https://github.com/gitcoinco/web/pull/9032/commits/17ac7e54422f8479b8ab97411936f98deba8b2ca